sdks/python/apache_beam/io/gcp/bigquery_file_loads_test.py

```diff
@@ -424,6 +424,87 @@ def test_records_traverse_transform_with_mocks(self):
       assert_that(jobs,
                   equal_to([job_reference]), label='CheckJobs')
 
+  @mock.patch('time.sleep')
+  def test_wait_for_job_completion(self, sleep_mock):
+    logging.getLogger().setLevel(logging.INFO)
```

Comment:
Reducing log level to INFO outputs unneccesary stuff like:
```
INFO:root:==================== <function fix_side_input_pcoll_coders at 0x7fbbfb2d3730> ====================
INFO:root:==================== <function lift_combiners at 0x7fbbfb2d37b8> ====================
INFO:root:==================== <function expand_sdf at 0x7fbbfb2d3840> ====================
INFO:root:==================== <function expand_gbk at 0x7fbbfb2d38c8> ====================
INFO:root:==================== <function sink_flattens at 0x7fbbfb2d39d8> ====================
INFO:root:==================== <function greedily_fuse at 0x7fbbfb2d3a60> ====================
INFO:root:==================== <function read_to_impulse at 0x7fbbfb2d3ae8> ====================
INFO:root:==================== <function impulse_to_input at 0x7fbbfb2d3b70> ====================
INFO:root:==================== <function inject_timer_pcollections at 0x7fbbfb2d3d08> ====================
INFO:root:==================== <function sort_stages at 0x7fbbfb2d3d90> ====================
INFO:root:==================== <function window_pcollection_coders at 0x7fbbfb2d3e18> ====================
INFO:root:Creating state cache with size 100
INFO:root:Created Worker handler <apache_beam.runners.portability.fn_api_runner.EmbeddedWorkerHandler object at 0x7fbbfb2927b8> for environment urn: "beam:env:embedded_python:v1"

INFO:root:Running (((ref_AppliedPTransform_assert_that/Create/Read_9)+(ref_AppliedPTransform_assert_that/Group/pair_with_0_13))+(assert_that/Group/Flatten/Transcode/0))+(assert_that/Group/Flatten/Write/0)
INFO:root:Running (ref_AppliedPTransform_job_ref/Read_3)+(ref_PCollection_PCollection_1/Write)
INFO:root:Running ((((((ref_AppliedPTransform_Create/Read_5)+(ref_AppliedPTransform_ParDo(WaitForBQJobs)_6))+(ref_AppliedPTransform_assert_that/WindowInto(WindowIntoFn)_10))+(ref_AppliedPTransform_assert_that/ToVoidKey_11))+(ref_AppliedPTransform_assert_that/Group/pair_with_1_14))+(assert_that/Group/Flatten/Transcode/1))+(assert_that/Group/Flatten/Write/1)
INFO:root:Job status: <Mock name='mock.status' id='140445338423704'>
INFO:root:Job status: <Mock name='mock.status' id='140445338424376'>
INFO:root:Job status: <Mock name='mock.status' id='140445338424992'>
INFO:root:Running (assert_that/Group/Flatten/Read)+(assert_that/Group/GroupByKey/Write)
INFO:root:Running (((assert_that/Group/GroupByKey/Read)+(ref_AppliedPTransform_assert_that/Group/Map(_merge_tagged_vals_under_key)_20))+(ref_AppliedPTransform_assert_that/Unkey_21))+(ref_AppliedPTransform_assert_that/Match_22)
```
Can this be removed if it's not being used in the test?

Reply:
great catch. thanks tanay!

Reply:
:)